bindings/python/test/test_bson.py

```diff
@@ -154,3 +154,24 @@ def test_object_id_type(self):
         arr = pa.ExtensionArray.from_storage(ObjectIdType(), storage_array)
         result = self.serialize_array(arr)
         assert result.type._type_marker == ObjectIdType._type_marker
+
+
+class TestBooleanType(TestBsonToArrowConversionBase):
+    def setUp(self):
+        self.schema = Schema({'_id': ObjectId,
```

Comment:
Can we remove the `_id` from this test to make is smaller and easier to read? IIUC _id+ObjectId serves no purpose in this test.

Reply:
Done.